src/main/java/com/upsolver/datasources/jdbc/querybuilders/DefaultQueryDialect.java

```diff
@@ -181,6 +194,25 @@ public boolean isAutoIncrementColumn(ResultSet columnsResultSet) throws SQLExcep
         return "yes".equalsIgnoreCase(columnsResultSet.getString("IS_AUTOINCREMENT"));
     }
 
+    @Override
+    public boolean isTimeType(SQLType sqlType) throws SQLException {
+        return timeTypes.contains(getJdbcType(sqlType));
+    }
+
+    @Override
+    public SQLType getSqlType(int code) throws SQLException {
+        try {
+            return JDBCType.valueOf(code);
+        } catch (IllegalArgumentException e) {
+            return new SimpleSqlType("unknown", "default", code);
```

Comment:
If this leads to reading the data successfully log warn. Otherwise throw error.

Reply:
added log